packages/flutter/lib/src/cupertino/form_section.dart

```diff
@@ -0,0 +1,316 @@
+  /// widgets be included in the [children] list in order to retain the iOS look.
+  final List<Widget> children;
+
+  /// Sets the decoration for the row group.
+  ///
+  /// If null, defaults to a conditional background color for the row group,
+  /// with [CupertinoColors.systemBackground] in light themes and
+  /// [CupertinoColors.secondarySystemBackground] in dark themes.
+  ///
+  /// If null, defaults to 10.0 circular radius when constructing with
+  /// [CupertinoFormSection.insetGrouped]. Defaults to zero radius for the
+  /// standard [CupertinoFormSection] constructor.
+  final BoxDecoration? groupDecoration;
+
+  /// Sets the background color for the section.
+  ///
+  /// If null, defaults to a conditional background color for the section, with
+  /// [CupertinoColors.secondarySystemBackground] in light themes and
+  /// [CupertinoColors.systemBackground] in dark themes.
+  final Color? backgroundColor;
+
+  /// {@macro flutter.material.Material.clipBehavior}
+  ///
+  /// Defaults to [Clip.none], and must not be null.
+  final Clip clipBehavior;
+
+  @override
+  Widget build(BuildContext context) {
+    final MediaQueryData media = MediaQuery.of(context);
+
+    final Color dividerColor = CupertinoColors.separator.resolveFrom(context);
+    final double dividerHeight = 1.0 / media.devicePixelRatio;
+
+    // Long divider is used for wrapping the top and bottom of rows.
+    // Only used in _CupertinoFormSectionType.base mode
+    final Widget longDivider = Container(
+      color: dividerColor,
+      height: dividerHeight,
+    );
+
+    // Short divider is used between rows.
+    // The value of the starting inset (15.0) is determined using SwiftUI's Form
+    // seperators in the iOS 14.2 SDK.
+    final Widget shortDivider = Container(
+      margin: const EdgeInsetsDirectional.only(start: 15.0),
+      color: dividerColor,
+      height: dividerHeight,
+    );
+
+    // We construct childrenWithDividers as follows:
+    // Insert a short divider between all rows.
+    // If it is a `_CupertinoFormSectionType.base` type, add a long divider
+    // to the top and bottom of the rows.
+    assert(children.isNotEmpty);
+
+    final List<Widget> childrenWithDividers = <Widget>[];
+
+    if (_type == _CupertinoFormSectionType.base) {
+      childrenWithDividers.add(longDivider);
+    }
+
+    children.sublist(0, children.length - 1).forEach((Widget widget) {
+      childrenWithDividers.add(widget);
+      childrenWithDividers.add(shortDivider);
+    });
+
+    childrenWithDividers.add(children.last);
+    if (_type == _CupertinoFormSectionType.base) {
+      childrenWithDividers.add(longDivider);
+    }
+
+    // When the background color is not provided, use the default background
+    // color determined from standard SwiftUI Forms in iOS 14.2 SDK.
+    // Defaults to black (systemBackground) in dark themes, and
+    // gray (secondarySystemBackground) in light themes.
+    final Color sectionBackgroundColor = backgroundColor ??
```

Comment:
Looks like iOS 14 is using a private color `tableCellGroupedBackgroundColor` for this. Could you create a corresponding private `CupertinoDynamicColor` for it?

Reply:
`CupertinoColors.systemGroupedBackground` and `CupertinoColors.secondarySystemGroupedBackground` were the right ones. Sorry about that.